test/unit/domain/thirdpartyRequests.test.ts

```diff
@@ -0,0 +1,109 @@
+ OF ANY KIND, either express or implied. See the License for the specific language governing
+ permissions and limitations under the License. Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file. Names of the original
+ copyright holders (individuals or organizations) should be listed with a '*' in the first column.
+ People who have contributed from an organization can be listed under the organization that actually
+ holds the copyright for their contributions (see the Gates Foundation organization for an example).
+ Those individuals should have their names indented and be marked with a '-'. Email address can be
+ added optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Kevin Leyow <kevin.leyow@modusbox.com>
+ --------------
+ ******/
+import {
+  buildPayeeQuoteRequestFromTptRequest,
+  forwardPostQuoteRequestToPayee,
+  verifyConsentId,
+  verifySourceAccountId,
+  verifyPispId,
+  validateGrantedConsent
+} from '~/domain/thirdpartyRequests/transactions'
+import TestData from 'test/unit/data/mockData.json'
+import { resetUuid } from 'test/unit/__mocks__/uuidv4'
+
+const mockData = JSON.parse(JSON.stringify(TestData))
+const postThirdpartyRequestsTransactionRequest = mockData.postThirdpartyRequestsTransactionRequest
+const postQuoteRequest = mockData.postQuotesRequest
+const __postQuotes = jest.fn(() => Promise.resolve())
+
+jest.mock('@mojaloop/sdk-standard-components', () => {
+  return {
+    MojaloopRequests: jest.fn(() => {
+      return {
+        postQuotes: __postQuotes
+      }
+    })
+  }
+})
+
+describe('thirdpartyRequests/transactions', (): void => {
+  beforeEach((): void => {
+    jest.clearAllMocks()
+    resetUuid()
+  })
+
+  it('buildPayeeQuoteRequestFromTptRequest should construct a proper quote request from tptRequest', (): void => {
+    const expectMockQuoteRequest = expect(buildPayeeQuoteRequestFromTptRequest(
+      postThirdpartyRequestsTransactionRequest.payload
+    ))
+
+    expectMockQuoteRequest.toEqual(expect.objectContaining({
+      quoteId: '00000000-0000-1000-8000-000000000001',
+      transactionId: '00000000-0000-1000-8000-000000000002',
+      note: ''
+    }))
+    expectMockQuoteRequest.toEqual(expect.objectContaining({
+      payee: postQuoteRequest.payload.payee,
+      payer: postQuoteRequest.payload.payer,
+      amount: postQuoteRequest.payload.amount,
+      amountType: postQuoteRequest.payload.amountType,
+      transactionType: postQuoteRequest.payload.transactionType
+    }))
+  })
+
+  it('forwardPostQuoteRequestToPayee should forward a quote request and forward it to a payee', (): void => {
+    // build expected quote request
+    const quoteRequest = buildPayeeQuoteRequestFromTptRequest(
+      postThirdpartyRequestsTransactionRequest.payload
+    )
+
+    resetUuid()
```

Comment:
could this be in a `beforeEach`?

Reply:
it is in before each. 

`forwardPostQuoteRequestToPayee` also uses the mock uuid. So I need to reset it here to make sure it sends a quote with matching uuids.